homework/01LessonOne.js

```diff
@@ -204,11 +207,11 @@ var stringFourStrictNotEqualsNumberFour;
 //
 // Your Answer Goes Here:
 //
-//
+// == is equal to so '8' == 8 is true because it is the same # even though one is a string. === is equal to the same value and type '8'=== 8 is false because they are not the same value. '8' is a string not a variable and 8 would be a variable.
 // What’s the difference between != and !==?
```

Comment:
"8" and 8 have the same value. **"==="** checks to equality in value AND **data type**.